machineLearning/sentimentAnalysis/bin/database/databaseFunctions.py

```diff
@@ -0,0 +1,55 @@
+import pymongo
+
+
+def getArticles(client,databaseName="test"):
+    """
```

Comment:
I know you're aware, but just need to make sure we update this (and all other references to the test DB) to the actual DB name when we changeover (likely next week after the interim presentation).

Reply:
Yeah can go through next week and update these, only local script the Lambda still working off parameter store so wont be as bad. Cheers!